website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -29,6 +33,608 @@ resource "newrelic_cloud_aws_link_account" "foo" {
+* `fetch_tags` - (Optional) Specify if tags should be collected. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `sqs`, `alb`, `elasticache`, `cloudfront`, `dynamodb`, `ecs`, `efs`, `elasticbeanstalk`, `elasticsearch`, `emr`, `kinesis`, `lambda`, `rds`
+* `queue_prefixes` - (Optional) Specify each name or prefix for the Queues that you want to monitor. Filter values are case-sensitive.
+  * the following integration blocks support the usage of this resource: `sqs`
+* `load_balancer_prefixes` - (Optional) Specify each name or prefix for the LBs that you want to monitor. Filter values are case-sensitive.
+  * the following integration blocks support the usage of this resource: `alb`
+* `stage_prefixes` - (Optional) Determine if extra inventory data be collected or not. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `api_gateway`
+* `fetch_lambdas_at_edge` - (Optional) Specify if Lambdas@Edge should be monitored. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `cloudfront`
+* `duplicate_ec2_tags` - (Optional) Specify if the old legacy metadata and tag names have to be kept, it will consume more ingest data size.
+  * the following integration blocks support the usage of this resource: `ec2`
+* `fetch_ip_addresses` - (Optional) Specify if IP addresses of ec2 instance should be collected.
+  * the following integration blocks support the usage of this resource: `ec2`
+* `fetch_shards` - (Optional) Specify if Shards should be monitored. May affect total data collection time and contribute to the Cloud provider API rate limit.
+  * the following integration blocks support the usage of this resource: `kinesis`
+
+
+All of these arguments may be used only with a certain set of arguments. In order to find the right set of arguments which go with each integration **check out the [Integration blocks](#integration-blocks) section below.**
+
+## Integration blocks
+
+The following lists out arguments which may be used with each AWS integration supported by this resource.
+
+<details>
+  <summary> Some integration types support an additional set of arguments. Expand this section to take a look at these supported arguments. </summary>
+  <details>
+  <summary>cloudtrail</summary>
+*  `aws_regions` `metrics_polling_interval`
+*  Valid values for metrics_polling_interval : 300, 900, 1800, 3600 (seconds)
+```hcl 
+     cloudtrail {
+        metrics_polling_interval = 300
+        aws_regions              = ["us-east-1", "us-east-2"]
+     }
+``` 
+  </details>
+  <details>
+  <summary>vpc</summary>
+  <ul>
```

Comment:
Please remove the <ul></ul> here to avoid the problem we're seeing with formatting - 

<img width="670" alt="image" src="https://github.com/user-attachments/assets/e84d7443-f829-4e26-af8f-944a63f35e68">